src/utils/limit.ts

```diff
@@ -0,0 +1,5 @@
+// Limit number within bounds.
+// Used ternary operator instead of `Math.min(Math.max(min, number), max)` to save few bytes
+export const limit = (number: number, min = 0, max = 1): number => {
```

Comment:
Imo, there is a better name for this method: `clamp`, used widely in animations (react-spring) and [the latest CSS proposal](https://developer.mozilla.org/en-US/docs/Web/CSS/clamp).

Reply:
Agree

Reply:
Done in 99be541